CITATION.cff

```diff
@@ -60,6 +60,34 @@ authors:
     family-names: Young
     affiliation: Queen's University Belfast
     orcid: 'https://orcid.org/0000-0002-1229-2499'
+  - given-names: Pedro
+    family-names: Bernardinelli
+    affiliation: University of Washington
+    orcid: 'https://orcid.org/0000-0003-0743-9422'
+  - given-names: Colin
+    family-names: Chandler
+    affiliation: University of Washington
+    orcid: ''
+  - given-names: Matthew
+    family-names: Holman
+    affiliation: Harvard University
```

Comment:
Center for Astrophysics — Harvard & Smithsonian
ORCID https://orcid.org/0000-0002-1139-4880